### PR TITLE
Added option to got to front page /new

### PR DESCRIPTION
**Pull request by tobywhughes**
Realized this was an easy thing to add. Just allows the user to do /r/new and bring up the front page's new.

### CODE REVIEW

**Comment by mekhami** [comment]
This functionality already exists with /r/front I believe.

File: rtv/content.py [hunk 1]
```diff
@@ -306,6 +306,9 @@ def from_name(cls, reddit, name, loader=default_loader):
 
         if name == 'all':
             sub = reddit.get_subreddit(name)
+
+        if name == 'new':
```
Comment (michael-lazar) [inline rtv/content.py]: This needs to be switched around, "all" should be attached to the else block.
```
if name == 'new':
    return cls('New', reddit.get_new(limit=None), loader)

if name == 'all':
    sub = reddit.get_subreddit(name)
else:
    ...
```

**Comment by michael-lazar** [comment]
Thanks for the pull request. @mekhami I think /r/front sorts the front page by 'hot', and this would sort by 'new'? Correct me if I'm wrong.

**Comment by mekhami** [comment]
Yeah, sorry, I think I was reading two different threads at once and got them mixed up. Completely my misunderstanding, sorry about that.

**Comment by tobywhughes** [comment]
Oh ya. My mind was thinking in else if's.

**Comment by tobywhughes** [comment]
There, fixed.

**Pull request merged**